interfaces/cython/cantera/test/test_reaction.py

```diff
@@ -486,6 +486,10 @@ class TestLindemannRate(FalloffRateTests, utilities.CanteraTest):
         high-P-rate-constant: {A: 7.4e+10, b: -0.37, Ea: 0.0}
         """
 
+    def test_falloff_function(self):
+        rate = self.from_parts()
+        assert rate.falloff_function(300, 0.) == 1.
```

Comment:
```suggestion
        assert np.allclose(rate.falloff_function(300, 0.0), 1.0)
```

I think floating point comparisons shouldn't use `==`, but maybe it's OK in this case.

Reply:
In this case, it's ok, as that specific falloff function is 1.0 by definition. But I agree in general. Let me know if you'd like to change it.

Reply:
The only concern I have is that `rate.falloff_function` returns `1.000000000001` as it goes through any calculations. If this isn't exercising any calculation, why have the test?

Reply:
I think `pytest` has something better than `numpy.allclose` here. How does
```python
assert rate.falloff_function(300, 0.) == pytest.approx(1, abs=1e-16)
```
look to you?

Reply:
Why is that better than `np.allclose`? There are three options here, `math.isclose(a, b)`, `pytest.approx(a, b)`, and `np.isclose/allclose(a, b)`. I think we should be consistent throughout the test suite which one we use, maybe this should use `self.assertNear` until (if) we switch to `pytest`-style asserts?

Reply:
> If this isn't exercising any calculation, why have the test?

I mainly added the test to ensure that the Python API is covered and won't break (again). The optional argument was left over from an earlier implementation that had the necessary infrastructure included.

Reply:
Ok. settling for `np.isclose`. I'd like to keep the test, and I'm actually not sure that we actually test for unity elsewhere.